php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -697,9 +697,15 @@ class NDB_BVL_Instrument extends NDB_Page
             if (strpos($key, '__') === 0) {
                 unset($values[$key]);
             }
+            if (strpos($key, '_status') !== false) {
+                Utility::nullifyEmpty($values, $key);
+            }
             if (array_key_exists($key, $this->WrapperNumericElements)) {
                 Utility::nullifyEmpty($values, $key);
             }
+            if (in_array($key, $this->dateTimeFields) && $values[$key] === '') {
+                unset($values[$key]);
```

Comment:
unset($values[$key]) isn't right because then the key won't be set to null in the update statement, but not be in the update statement at all (which means it'll retain the old value instead of setting it to null when the empty string is provided.

This should be using the same Utility::nullifyEmpty() function for consistency between the three statements (which means these three if statements could also become one if statement with an or, if you wanted....)